src/components/MAPI/utils.ts

```diff
@@ -230,6 +233,24 @@ export function fetchProviderNodePoolsForNodePoolsKey(
   return keys.join();
 }
 
+export async function fetchCluster(
```

Comment:
Will refactor other places to use this function for fetching CAPI cluster resources, so we can have different versions per provider.